pages/api/workshop.ts

```diff
@@ -0,0 +1,64 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { table, base } from "../../utils/table"
+import IEvent from '../../utils/IEvent'
+
+const filterStrings = [
+  'Test Event',
+  'The Ultimate Future Event',
+  'Hack Night'
+]
+
+const handler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+) => {
+  const { method } = req;
+  switch (method) {
+    case 'GET':
+      const events:IEvent[] = [];
```

Comment:
You don't need to populate a new array. Instead, when you call `table.select`, you can pass in `maxRecords: 3` (not 100% sure of the syntax), and it will return an array of 3 events, which you can send directly in the API response.